app/styles/_code.scss

```diff
@@ -28,7 +19,14 @@ code {
   padding: 4px 4px 2px 4px;
 }
 
-.light {
+.code--theme-dark {
+  pre, code {
```

Comment:
I think you can nest this under the other `.code--theme-dark` rule below, rather than having 2